src/test/data/XmlSerializableLoanBookTest/typicalLoansLoanBook.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!-- AddressBook save file which contains the same Loan values as in TypicalLoans#getTypicalAddressBook() -->
-<addressbook>
+<!-- LoanBook save file which contains the same Loan values as in TypicalLoans#getTypicalAddressBook() -->
```

Comment:
Comment still says ` TypicalLoans#getTypicalAddressBook()` instead of `TypicalLoans#getTypicalLoanBook()`